lib/classes/cart_checkout.rb

```diff
@@ -91,4 +91,8 @@ def designate_package(package, order)
   def add_errors(errs)
     errs.full_messages.each { |m| errors.add(:base, m) }
   end
+
+  def active_orders
```

Comment:
I think the `order` model already has lists of states (by type)
Maybe you can use those directly (e.g `Order::ACTIVE_STATES`) to centralise the types of states